src/foam/color/colorlib.js

```diff
@@ -196,10 +196,34 @@ foam.LIB({
       };
     },
     
-    function getBestForeground (str, black, white) {
+    function getBestForeground(str, black, white) {
       with ( foam.Color.parse(str) ) {
-        return red*0.3 + green*0.6 + blue*0.1 > 170 ? black : white;
+        return red*0.299 + green*0.587 + blue*0.114 > 186 ? black : white;
       }
-    }
+    },
+
+    function lighten(str, value) {
+      colorObj = foam.Color.parse(str);
+      var [r, g, b] = [colorObj.red, colorObj.green, colorObj.blue];
+      let l = foam.Color.rgbToGrey([r, g, b])*1000;
```

Comment:
maybe lib functions should take a `colorObj` as their input instead of `[r, g, b]`